api/external/sms.html

```diff
@@ -320,6 +320,36 @@ <h1>SMS<a class="headerlink" href="#sms" title="Permalink to this headline">¶</
                             </div>
                         </dd>
                     </dl>
+                    <dl class="get">
+                        <dt id="get--#account_id-sms-billing-credit-usage">
+                            <tt class="descname">GET </tt><tt class="descname">/#account_id/sms/billing/credit_usage</tt><a class="headerlink" href="#get--#account_id-sms-billing-credit-usage" title="Permalink to this definition">¶</a></dt>
+                        <dd><p>Get sms credit usage for account.</p>
+                            <p>This endpoint will return the credit usage for the account ID. You must specify query parameters 'from_date' and 'to_date'.</p>
```

Comment:
The only thing I'd want to add is noting that the date range is inclusive of the start date and exclusive of the end date. We probably should also note that the dates are based around the UTC timezone. Maybe something like this? We could alternatively update the parameters docs below. I'm not quite sure on the exact copy here, but maybe something like this

```suggestion
                            <p>This endpoint will return the credit usage for the account ID. You must specify query parameters 'from_date' and 'to_date'. The returned credit usage is inclusive of the the 'from_date' and exclusive of the 'from_date'. The dates are based in the UTC timezone.</p>
```